File: test/fixtures/node-axios/j_patch_file_only.js
import axios from 'axios';
import FormData from 'form-data';
import { fs } from 'fs';

const form = new FormData();
form.append('file1', fs.readFileSync('./test/fixtures/curl_commands/delete.sh'), './test/fixtures/curl_commands/delete.sh');

const response = await axios.patch(
  'http://localhost:28139/patch',
  form,
  {
    headers: {
      ...form.getHeaders()
    }
  }
);

File: test/fixtures/java-unirest/j_patch_file_only.java
import java.io.File;
import java.io.FileInputStream;
import java.io.FileNotFoundException;
import kong.unirest.HttpResponse;
import kong.unirest.Unirest;

class Main {
    public static void main(String[] args) throws FileNotFoundException {
            HttpResponse<String> httpResponse
               = Unirest.patch("http://localhost:28139/patch")
            .field("file1", new FileInputStream(new File("./test/fixtures/curl_commands/delete.sh")), "./test/fixtures/curl_commands/delete.sh")
            .asString();
    }
}
